docs/pom.xml

```diff
@@ -207,6 +207,11 @@
             <artifactId>postgresql</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
is this needed? I don't see any actual code references in the doc folder.

Reply:
Strange, I had a build failure about docs project dependencies at some point, but I don't see it now.

Reply:
@hgschmie Yes, it is, otherwise docs cannot build:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.11.2:aggregate (aggregate) on project jdbi3-docs: An error has occurred in Javadoc report generation: 
Error:  Exit code: 1
Error:  /home/runner/work/jdbi/jdbi/postgres/src/main/java/org/jdbi/v3/postgres/PostgresPlugin.java:20: error: package com.pgvector does not exist
Error:  import com.pgvector.PGbit;
```